README.md

```diff
@@ -1 +1,11 @@
 # LibreJargon
+An open-source jargon demystifier.
+
+## Features:
+- Builtin PDF viewer
```

Comment:
We should expand on all of these, and maybe have a section about future things we would do?